zcash_primitives/src/transaction/fees/zip317.rs

```diff
@@ -20,6 +20,12 @@ use crate::{
 /// [ZIP 317]: https//zips.z.cash/zip-0317
 pub const MINIMUM_FEE: Amount = Amount::const_from_i64(10_000);
 
+/// The marginal fee using the standard [ZIP 317] constants. Equivalent to
+/// `FeeRule::standard().marginal_fee()`, but as a constant.
```

Comment:
```suggestion
/// The marginal fee using the standard [ZIP 317] constants. 
```

Reply:
`FeeRule::standard` could change at some point and this would then be incorrect.